package.json

```diff
@@ -19,6 +19,8 @@
     "express": "^4.17.1",
     "express-async-errors": "^3.1.1",
     "jsonwebtoken": "^8.5.1",
+    "jwt-decode": "^2.2.0",
```

Comment:
we removed this library from the project, please pull new dev branch

Reply:
Done